packages/matter.js/src/behavior/system/commissioning/CommissioningBehavior.ts

```diff
@@ -164,7 +163,7 @@ export namespace CommissioningBehavior {
         passcode = -1;
         discriminator = -1;
         readonly flowType = CommissioningFlowType.Standard;
-        readonly additionalBleAdvertisementData?: ByteArray;
-        readonly automaticAnnouncement = true;
+        readonly additionalBleAdvertisementData = new ByteArray(); // TODO Needed else can not be set!
```

Comment:
Hmm, should we remove readonly from these?

Reply:
If the current "readonly" means that they are bound to this one value here then Yes we need to because else it will never be configurable. If the config simply overwrites them and "readonly" just makes sure that noone changes the values after initialzzation with cinfig then we should leave it that way :-)